kvrocks.conf

```diff
@@ -530,22 +530,36 @@ migrate-sequence-gap 10000
 
 ################################ ROCKSDB #####################################
 
+# Specify the capacity of column family block cache. A larger block cache
+# may make requests faster while more keys would be cached. Max Size is 400*1024.
+# Default: 4096MB
+rocksdb.block_cache_size 4096
+
+# If this configuration is set to yes, the following block cache settings
+# (rocksdb.metadata_block_cache_size, rocksdb.subkey_block_cache_size,
+# rocksdb.share_metadata_and_subkey_block_cache) will be ignored, only using
+# rocksdb.block_cache_size to configure.
+# If this configuration is set to no, the following block cache settings
+# should be set. (deprecated)
+# Default: yes
+rocksdb.set_block_cache_size yes
+
 # Specify the capacity of metadata column family block cache. A larger block cache
 # may make requests faster while more keys would be cached. Max Size is 200*1024.
 # Default: 2048MB
-rocksdb.metadata_block_cache_size 2048
+#rocksdb.metadata_block_cache_size 2048 (deprecated)
```

Comment:
It is not good to write `(deprecated)` here. Since it is not a vaild config item then.